bin/configs/kotlin-spring-boot-request-cookie.yaml

```diff
@@ -0,0 +1,9 @@
+generatorName: kotlin-spring
+outputDir: samples/server/petstore/kotlin-springboot-request-cookie
```

Comment:
i ran `gradle build` on this folder but got lots of errors.

did you get errors when trying to build the project locally?

Reply:
I tried to run the project and my local gradle isn't even able to create the tasks.. i tried the same with the other samples but got the same result so i wasn't sure if that was expected, maybe it's a gradle version issue. I created this PR and wait to see if that was an issue or just something to ignore.

Reply:
I ran `mvn clean package` in this output folder but got some errors:
```
\org\openapitools\api\AnotherFakeApi.kt: (42, 19) Unresolved reference: another
[ERROR] C:\Users\wing3\Code\openapi-generator\samples\server\petstore\kotlin-springboot-request-cookie\src\main\kotlin\org\openapitools\api\FakeApi.kt: (422, 126) Parameter name expected
[ERROR] C:\Users\wing3\Code\openapi-generator\samples\server\petstore\kotlin-springboot-request-cookie\src\main\kotlin\org\openapitools\api\FakeApi.kt: (422, 127) Expecting comma or ')'
[ERROR] C:\Users\wing3\Code\openapi-generator\samples\server\petstore\kotlin-springboot-request-cookie\src\main\kotlin\org\openapitools\api\FakeApi.kt: (422, 127) Expecting ')'

```

can you please take a look?

`mvn clena package` works fine with the master using the config/spec you provided in this PR

Reply:
I've got different errors but i figured what was causing them, the spec file that's being used is a copy from a `fake-petstore` file for version 3 which has some cases that are not properly handled in `kotlin-spring` generator unless the configuration for the generator is not correct for those cases.

I've removed all the fake cases except the cookie one in the spec and now the sample doesn't throw any error from my end, feel free to run the tasks again and let me know if there are any other issues.